### ISSUE TITLE
TypeScript 3 RC? "ts.unescapeIdentifier is not a function"

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [ajafff/tsutils#71]
```
The 'no-misused-new' rule threw an error in 'D:/Code/WebClient/src/typings/lib.dom.custom.d.ts':
TypeError: ts.unescapeIdentifier is not a function
    at getIdentifierText (D:\Code\WebClient\node_modules\tsutils\util\util.js:872:15)
    at Object.getPropertyName (D:\Code\WebClient\node_modules\tsutils\util\util.js:208:61)
    at cb (D:\Code\WebClient\node_modules\tslint\lib\rules\noMisusedNewRule.js:51:27)
    at visitNodes (D:\Code\WebClient\node_modules\typescript\lib\typescript.js:15098:30)
    at Object.forEachChild (D:\Code\WebClient\node_modules\typescript\lib\typescript.js:15404:21)
    at cb (D:\Code\WebClient\node_modules\tslint\lib\rules\noMisusedNewRule.js:67:19)
    at visitNodes (D:\Code\WebClient\node_modules\typescript\lib\typescript.js:15098:30)
    at Object.forEachChild (D:\Code\WebClient\node_modules\typescript\lib\typescript.js:15324:24)
    at walk (D:\Code\WebClient\node_modules\tslint\lib\rules\noMisusedNewRule.js:49:15)
```

**Comment by ajafff** [comment]
Are you sure you are using the latest version? This should be fixed in v2.27.2

**Comment by JoshuaKGoldberg** [comment]
Apologies, it is!
